src/fswAlgorithms/attControl/thrMomentumManagementCpp/thrMomentumManagementCpp.h

```diff
@@ -28,6 +28,8 @@
 #include "architecture/msgPayloadDefC/RWSpeedMsgPayload.h"
 #include "architecture/msgPayloadDefC/CmdTorqueBodyMsgPayload.h"
 
+#include "Eigen/Dense"
```

Comment:
:large_orange_diamond: Including Eignen/Core was probably better due to it's narrower scope. Was there a reason to include the broader Eigen/Dense?

Reply:
Not really, I'll switch back to Eigen/Core.

Reply:
While you don't have to adjust, the best way to have reverted his change would have been to rebase and edit the commit to remove the change all together.